riscv-c-api.md

```diff
@@ -33,28 +33,41 @@ https://creativecommons.org/licenses/by/4.0/.
 
 ## Preprocessor Definitions
 
-* `__riscv`
-* `__riscv_abi_rve`
-* `__riscv_32e`
-* `__riscv_xlen`
-* `__riscv_flen`
-* `__riscv_atomic`
-* `__riscv_compressed`
-* `__riscv_mul`
-* `__riscv_div`
-* `__riscv_muldiv`
-* `__riscv_fdiv`
-* `__riscv_fsqrt`
-* `__riscv_float_abi_soft`
-* `__riscv_float_abi_single`
-* `__riscv_float_abi_double`
-* `__riscv_float_abi_quad`
-* `__riscv_cmodel_medlow`
-* `__riscv_cmodel_medany`
-* `__riscv_cmodel_pic`
-* `__riscv_vector`
-* `__riscv_bitmanip`
-* `__riscv_zfh`
+| Name                | Value | When defined                  |
+| ------------------- | ----- | ----------------------------- |
+| __riscv             | 1     | Always defined.               |
+| __riscv_xlen        | <ul><li>32 for rv32</li><li>64 for rv64</li><li>128 for rv128</ul> | Always defined.             |
+| __riscv_flen        | <ul><li>128 if `Q` extension available</li><li>64 if `D` extension available</li><li>32 if `F` extension available<li></ul> | `F` extension is available. |
```

Comment:
Or rather:

- 32 if the `F` extension is available **or**
- 64 if the `D` extension is available **or**
- 128 if the `Q` extension is available

Reply:
Updated, thanks :)